recipes/gperf/all/conandata.yml

```diff
@@ -2,6 +2,9 @@ sources:
   "3.1":
     url: "https://ftp.gnu.org/pub/gnu/gperf/gperf-3.1.tar.gz"
     sha256: "588546b945bba4b70b6a3a616e80b4ab466e3f33024a352fc2198112cdbb3ae2"
+  "cci.20210606":
+    url: "https://git.savannah.gnu.org/gitweb/?p=gperf.git;a=snapshot;h=96ebd0b95dda2817923ef6de25cdaca1e7ee2956;sf=tgz"
+    sha256: "e9b40d8d573e67db145a726a762f2a031cd64a300f8aa5cc3d576ee8f1ea93f5"   
```

Comment:
Is this guaranteed to always create the exact same download?

Reply:
@Croydon I have no idea. Here is where I got the snapshot from:
https://git.savannah.gnu.org/gitweb/?p=gperf.git;a=commit;h=96ebd0b95dda2817923ef6de25cdaca1e7ee2956

Reply:
Today, 3 days later, the hash is still the same.

Reply:
@madebr For sure if you look at this branch https://github.com/conan-io/conan-center-index/pull/4332/files from 6 months ago the link still produces the same hash. I was just saying that I don't see any mention of a guarantee on their repo and I don't know if they provide such a guarantee.

Reply:
Yup, this is the best we can do + pray that gitweb does not change their archive library :smile: